src/index.ts

```diff
@@ -33,18 +37,91 @@ joplin.plugins.register({
 		// doesn't always work when first opening the app. Opening the keyboard
 		// shortcuts will properly bind it and make it work.
 		// Placing the command before registering settings also seems to fix it
+
+		// TODO: no longer needed?
+
+		// await joplin.commands.register({
+		// 	name: 'editor.richMarkdown.clickAtCursor',
+		// 	label: 'Perform action',
+		// 	iconName: 'fas fa-link',
+		// 	execute: async () => {
+		// 		await joplin.commands.execute('editor.execCommand', {
+		// 			name: 'clickUnderCursor',
+		// 		});
+		//   },
+		// });
```

Comment:
This is still needed to bind Ctrl+Enter to the "Perform Action" command, which will follow a link or toggle a checkbox. It's a handy command that I use fairly often (mostly to toggle checkboxes).

Reply:
Right, I wasn't quite sure about this. I've restored the command now.